src/components/atoms/Input/types.ts

```diff
@@ -13,11 +13,12 @@ export interface InputProps {
   helperText?: string
   rightAdornment?: ReactNode | string
   Icon?: ReactNode
-  iconSize?: IconSize
+  iconSize?: IconSize,
+  onChange: (e: string) => void
+  validator?: (e: string) => string,
+  value?: any,
```

Comment:
@raulvillca can you update this part to use a union type instead of any.

eg:
value: string | number